src/models/Likes_DislikesModel.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+declare(strict_types=1);
+
+class LikesModel
+{
+    public PDO $db;
+
+    public function __construct(PDO $db)
+    {
+        $this->db = $db;
+    }
+
+    public function trackActivity(int $postsId, int $usersId): array
```

Comment:
This function name is a bit vague about what it does. Hard to think of something better, but you should at least say it's about Liking / Disliking? 

If we think about what this is doing, it's getting the array of all like/dislike responses the user has made. So the function is really doing 'get Likes/Dislikes', except you haven't recorded which rows are likes and which are dislikes in that table.

Tricky. We can leave it as is and call it technical debt.